src/style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars__1 .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars__2 .stars__star:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars__3 .stars__star:nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars__4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars__5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
This can be simplified, look at checklist point 2 and group them

Reply:
```
.stars--1 .stars--star:first-child,
.stars--2 .stars--star:nth-child(-n + 2),
.stars--3 .stars--star:nth-child(-n + 3),
.stars--4 .stars--star:nth-child(-n + 4),
.stars--5 .stars--star:nth-child(-n + 5) {
  background-image: url(images/star-active.svg);
}
```

Like this?